website/templates/emails/confirm_collections.html.mako

```diff
@@ -0,0 +1,17 @@
+<%inherit file="notify_base.mako" />
+
+<%def name="content()">
+<tr>
+  <td style="border-collapse: collapse;">
+    Hello ${user.fullname},<br>
+    <br>
+    Welcome to ${provider}, powered by the Open Science Framework. To continue, please verify your email address by visiting this link:<br>
```

Comment:
Verifying that Product wanted a specific collections-related email and not just a generic OSF one?